modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/atomic/GridDhtAtomicCache.java

```diff
@@ -674,11 +674,18 @@ private IgniteInternalFuture<Map<K, V>> getAllAsyncInternal(
 
     /** {@inheritDoc} */
     @Override public IgniteInternalFuture<?> putAllConflictAsync(Map<KeyCacheObject, GridCacheDrInfo> conflictMap) {
+        if (F.isEmpty(conflictMap))
```

Comment:
Why do you need this if?

Reply:
I used this 'if' to avoid processing empty maps. The same logic is used for putAll methods

Reply:
Removed it